libraries/Microsoft.Bot.Builder/TelemetryLoggerMiddleware.cs

```diff
@@ -46,6 +59,15 @@ public TelemetryLoggerMiddleware(IBotTelemetryClient telemetryClient, bool logPe
         [JsonIgnore]
         public IBotTelemetryClient TelemetryClient { get; }
```

Comment:
Should this be obsolete now?

Reply:
marking it as obselete to indicate that a user that they should update their config but leaving in place so that existing solutions still work